rocky/katalogus/templates/partials/plugin_tile.html

```diff
@@ -30,18 +30,23 @@
         {% endif %}
         <span class="plugin-author-link de-emphasized"><a target="_blank" href="https://openkat.nl" rel="noopener noreferrer">OpenKAT</a></span>
         {% if show_report_types %}
-            <div class="tags">
-                <p>
-                    <strong>{% translate "Required for:" %}</strong>
-                </p>
-                {% for plugin_id, report_types in plugin_report_types.items %}
-                    {% if plugin_id == plugin.id %}
-                        {% for report_type in report_types %}
-                            <span class="label tags-color-{{ report_type.label_style }}">{{ report_type.name }}</span>
-                        {% endfor %}
+            {% for plugin_id, report_types in plugin_report_types.items %}
+                {% if plugin_id == plugin.id %}
+                    {% if report_types|length >= 2 %}
+                        {% include "partials/modal_report_types.html" with plugin=plugin %}
+
+                    {% else %}
+                        <div class="tags">
+                            <p>
+                                <strong>{% translate "Required for: " %}</strong>
+                            </p>
+                            {% for report_type in report_types %}
```

Comment:
Same here, I think. I know you only moved this, but since it's visually clearly a list of distinct items, it should be so semantically as well.